install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,279 @@ by performing the following steps:
+|Selector for the nodes on which Prometheus is deployed.
+
+|`openshift_prometheus_storage_kind`
+|Set to create PV for Prometheus. For example,
+`openshift_prometheus_storage_kind=nfs`.
+
+|`openshift_prometheus_alertmanager_storage_kind`
+|Set to create PV for Alertmanager. For example,
+`openshift_prometheus_alertmanager_storage_kind=nfs`.
+
+|`openshift_prometheus_alertbuffer_storage_kind`
+|Set to create PV for Alert Buffer. For example,
+`openshift_prometheus_alertbuffer_storage_kind=nfs`.
+
+|`openshift_prometheus_storage_type`
+|Set to create PVC for Prometheus. For example,
+`openshift_prometheus_storage_type=pvc`.
+
+|`openshift_prometheus_alertmanager_storage_type`
+|Set to create PVC for Alertmanager. For example,
+`openshift_prometheus_alertmanager_storage_type=pvc`.
+
+|`openshift_prometheus_alertbuffer_storage_type`
+|Set to create PVC for Alert Buffer. For example,
+`openshift_prometheus_alertbuffer_storage_type=pvc`.
+
+|`openshift_prometheus_additional_rules_file`
+|Additional Prometheus rules file. Set to `null` by default.
+
+|===
+
+[[openshift-prometheus-deploy]]
+=== Deploying Prometheus Using Ansible Installer
+
+The Ansible Installer is the default method of deploying Prometheus.
+
+Add label to your node:
+----
+# Inventory file
+openshift_prometheus_namespace=openshift-metrics
+
+openshift_prometheus_node_selector={"region":"infra"}
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+[[openshift-prometheus-additional-deploy]]
+==== Additional Methods for Deploying Prometheus
+
+*Deploy Using Node-Selector*
+
+Label the node on which you want to deploy Prometheus:
+----
+# oadm label node/$NODE ${KEY}=${VALUE}
+----
+
+Deploy Prometheus with Ansible and container resources:
+----
+# Inventory file
+openshift_prometheus_namespace=openshift-metrics
+
+# Set node selector for prometheus
+openshift_prometheus_node_selector={"${KEY}":"${VALUE}"}
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+*Deploy Using a Non-default Namespace*
+
+Identify your namespace:
+----
+# Inventory file
+openshift_prometheus_node_selector={"region":"infra"}
+
+# Set non-default openshift_prometheus_namespace
+openshift_prometheus_namespace=${USER_PROJECT}
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+[[openshift-prometheus-web]]
+==== Accessing the Prometheus Web UI
+
+The Prometheus server automatically exposes a Web UI at `localhost:9090`. You
+can access the Prometheus Web UI with the `view` role.
+
+[openshift-prometheus-config]
+==== Configuring Prometheus for OpenShift
+
+Example Prometheus rules file:
+----
+# additional prometheus rules file
+openshift_prometheus_additional_rules_file: null
+
+# All the required exports
```

Comment:
@bmcelvee 
line 914-977 is not example Prometheus rules file, these contents are related to PVs, I think you put them in wrong place

Reply:
@juzhao Do you think it would be best to remove this content from the documentation? Or move it elsewhere? I will comment it out in the follow-up PR for the time being. 

Perhaps directing readers here would be better: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/defaults/main.yaml

Reply:
@bmcelvee They are Prometheus storage configs, I think your put these configs in mail.yaml is find